erpnext/assets/doctype/asset_repair/asset_repair.py

```
@@ -411,33 +424,153 @@ def get_downtime(failure_date, completion_date):
 
 
 @frappe.whitelist()
+@frappe.validate_and_sanitize_search_inputs
 def get_purchase_invoice(doctype, txt, searchfield, start, page_len, filters):
```

Ruff (0.14.6)

Unused function argument: `doctype`

(ARG001)